packages/esm-patient-test-results-app/src/hiv/concepts.ts

```diff
@@ -0,0 +1,58 @@
+const concepts = {
+  display: 'Bloodwork',
+  concept: 'ConvSet', // convienence set
+  obs: [],
+  subSets: [
+    {
+      display: 'Hematology',
+      concept: 'ConvSet',
+      obs: [],
+      subSets: [
+        {
+          display: 'Complete Blood Count',
+          concept: 'LabSet',
+          obs: [
+            {
+              display: 'Hematocrit',
```

Comment:
Would you want the `value` here?

Reply:
In addition to the value, do you also need the `obsDatetime`?

Reply:
Yes, value would go here. I am not familiar yet with the observation data structure, but all data necessary to populate the timeline would need to be included here.

Reply:
I do not want this to be based on the observation data structure (which is not client friendly in the first place), but rather on what you need to accomplish your task. 😊

Reply:
We'd probably need: `obsDateTime`, `value`, `interpretation` at a minimum. Possibly `encounter` and `status` (where it exists). IIRC, the current timeline is built on top of the FHIR API. The main difference there is we provide the the reference ranges (from the concept) along-side the actual value (which is important for the visual indicators generated for concerning results).

Reply:
@dkayiwa  Let me start to de-structure the current timeline so I can tell you exactly what fields will be needed here.

Reply:
+1 to @ibacher's list. In addition...

For the concept (row):
* A uuid or url to the obs concept (you may only need a localized name for display, but concept names – especially short names for use in tables – are not unique)
* `units` (already part of the requirements for data to be displayed)

For the obs (values):
* `comment` (if not null) should always be included and shown (via footnote, tooltip, or other drill-down), since it may contain 
* A uuid or url to the obs, so it's straightforward to get additional detail of the obs when a user interacts with a specific value in the table

Reply:
We will refine the endpoint and endpoint contents on subsequent PR. For this PR - FilterSet component I think it is ok to proceed without this being finalized. Thanks for the initial endpoint @dkayiwa